### PR TITLE
commandbox 5.0.0

### PR DESCRIPTION
**Pull request by bdw429s**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/commandbox.rb [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
   homepage "https://www.ortussolutions.com/products/commandbox"
-  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/4.8.0/commandbox-bin-4.8.0.zip"
-  sha256 "43ddb16d221b5e9e7165349195c3b1c3714b28710ba21d88d4cb6ad650213861"
+  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/5.0.0/commandbox-bin-5.0.0.zip"
+  sha256 "1f8a355c8851f35944f58c96a18ca5eb9d9f14364ebb61c1269149cefa1efa0b"
 
   bottle :unneeded
 
-  depends_on :java => "1.8"
+  depends_on :java => ">= 8"
```
Comment (fxcoudert) [inline Formula/commandbox.rb]:
```suggestion
  depends_on :java => "1.8+"
```

**Comment by bdw429s** [comment]
Thx for the suggestion @fxcoudert, I've made that change.  Can you point me to the documentation for how to specify Java versions in my formula?  I must say I've been frustrated at the lack of information I've been able to find.  My original commit was actually something I copied and pasted out of a blog online and this is the second time I've had to update it because it wasn't "good enough".  it would have been nice to have a reference to show me how to do it right the first time.

**Comment by Bo98** [comment]
> how to specify Java versions in my formula

To be honest, the answer now will be you just `depend_on "openjdk"` (or `depend_on "openjdk@11" if necessary). `depends_on :java` is slowly being phased out (except for those that don't work in OpenJDK 11 and later) but that's not yet happened to all formulae.

> I copied and pasted out of a blog online and this is the second time I've had to update it because it wasn't "good enough"

I know you're frustrated, but I'm unfamiliar with which blog that is. That syntax hasn't ever worked to my knowledge.

Pull request reviews are not just us trying to beat down what you've done - as maintainers, we even review pull requests from other maintainers.

**Comment by bdw429s** [comment]
Thanks for the additional information.  I like the look of the new syntax.  I'm not a Ruby dev so it's already a bit confusing to navigate the Ruby stuff going on in there.  Please let me know if there are any other changes needed for this pull.

**Comment by bdw429s** [comment]
Thanks again for the help guys.

**Pull request merged**